pod-node/README.md

```diff
@@ -0,0 +1,21 @@
+# Pod-Node
+
+It's a native node module, wraps the pod-library.
+
+# Prerequisites
+
+After everything is ready with SGX _(check "pod-server" folder README)_, be sure you have;
```

Comment:
I think this should be `pod-client` and not `pod-server` :-)

Reply:
Ohm, right. my bad. It's a quick draft anyway. Will fix it, thanks :)